### PR TITLE
Chore/fix post contacts example

### PR DESCRIPTION
**Pull request by proudlygeek**
## PR Description

This PR adds several issues in the current docs:

- Snippets for `POST`,  `PUT` and `PATCH` don't contain the actual body of the request
- Snippet are non indented consistently
- Fix an issue with a banner not being properly closed
- Removes the UUID param on `POST /contacts` (reported by a confused user)

**Before:**

image

**After:**

image

### CODE REVIEW

File: docs/api_reference/contacts_api/patch_contacts.md [hunk 1]
```diff
@@ -29,7 +29,9 @@ Updates an existing contact.
 
 :::caution
 Ensure that `custom_fields` and `tags` already exist in your account before passing them. Visit [tags](https://dash.callbell.eu/settings/tags) and [custom_fields](https://dash.callbell.eu/settings/custom_fields) in your settings for more information.
+
 Similarly, for `assigned_user`, use a valid email address associated with a user in your account.
+:::
```
Comment (proudlygeek) [inline docs/api_reference/contacts_api/patch_contacts.md:34]: Note: closing the banner here.

File: src/snippets/curl/contacts_api/_patch_contacts.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ```bash
   curl -X PATCH "https://api.callbell.eu/v1/contacts/414a6d692bd645ed803f2e7ce360d4c8" \
       -H "Authorization: Bearer test_gshuPaZoeEG6ovbc8M79w0QyM" \
-      -H "Content-Type: application/json"
+      -H "Content-Type: application/json" \
```
Comment (proudlygeek) [inline src/snippets/curl/contacts_api/_patch_contacts.mdx:4]: Note: not sure on how was supposed to work but this is not valid cURL syntax 🤔 I had to add the missing backslash to make the script work.

Comment (Qwertic) [reply src/snippets/curl/contacts_api/_patch_contacts.mdx:4]: Oh nice, not sure why it was missing, thanks for fixing it 🙏🏼

**Review by Qwertic — APPROVED** [review]
LGTM 💯

**Review by camilamiz — APPROVED** [review]
🚀

**Pull request merged**